core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -148,6 +173,79 @@ oppia.directive('tutorCard', [
             });
           };
 
+          $scope.playPauseAudioTranslation = function() {
+            // TODO(tjiang11): Change from on-demand loading to pre-loading.
+
+            if (cardIndexOfPlayingAudio === undefined) {
```

Comment:
Thinking ahead, this will be insufficient. We will probably want audio for feedback, hints, etc.

Is there a more unique identifier we could use?